src/components/CrimCodeGridList.js

```diff
@@ -45,4 +42,34 @@ const CrimCodeGridList = props => {
   );
 };
 
-export default CrimCodeGridList;
+export const CrimCodeSectionList = props => {
+  const navAid = useNavigation();
+  return (
+    <View>
+      <Pressable
+        onPress={() =>
+          navAid.navigate('ContentCCScreen', {heading2key: props.heading2key})
+        }>
+        <View>
+          <View style={styles.gridListItem}>
+            <View style={[styles.gridItemMargin, styles.body]}>
+              <Text style={styles.heading2labelText}>
+                {props.firstSectionLabel}
+              </Text>
+              <Text style={styles.heading2TitleText}>
+                {props.heading2titletext}
+              </Text>
+            </View>
+
+            <Icon
+              style={styles.partsIcon}
+              name={'arrow-forward-ios'}
+              size={18}
```

Comment:
Can we decide on the size of these Icons? I have 20 up top, and 18 here. 20 on LegislationGridList too.

Reply:
Fixed.